pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,833 @@
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+The location of the USWDS source files is different in USWDS 3.0. You'll need to update your compiler settings to compile your Sass from USWDS 3.0.
+
+<div class="usa-accordion usa-accordion--bordered">
+  <!-- Use the accurate heading level to maintain the document outline -->
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a6"
+    >
+      If you're using USWDS Gulp
+    </button>
+  </h4>
+  <div id="m-a6" class="usa-accordion__content usa-prose">
+    <ol>
+      <li>Search for a line like <strong>const USWDS = "node_modules/uswds/dist"</strong> in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.</li>
+      <li>
+        Update the USWDS <strong>const</strong> to the updated uswds package location
+        <p><strong>Old</strong></p>
+          <code>const USWDS = "node_modules/uswds/dist";</code>
+        <p><strong>New</strong></p>
+          <code>const USWDS = "node_modules/@uswds/uswds";</code>
+          <br>
+      </li>
+      <li> 
+        Search for <strong>includePaths</strong> in your project's Gulp files. The paths in this list are where the Sass compiler looks for your source files. In USWDS 3.0
+        <p><strong>Old</strong></p>
+        <pre>
+          <code>.pipe(
+              sass({
+                includePaths: [
+                  PROJECT_SASS_SRC,
+                  `${USWDS}/scss`,
+                  `${USWDS}/scss/packages`,
+                ],
+              })
+            )</code>
+        </pre>
+        <p><strong>New</strong></p>
+        <pre>
+          <code>
+            .pipe(
+              sass({
+                includePaths: [
+                  PROJECT_SASS_SRC,
+                  `${USWDS},
+                  `${USWDS}/packages`,
+                ],
+              })
+            )
+          </code>
+        </pre>
+      </li>
+      <li>Recompile your Sass as usual. When it compiles, it is now using USWDS 3.0!</li>
+    </ol>
+  </div>
+
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a7"
+    >
+      If you're using USWDS Compile
+    </button>
+  </h4>
+  <div id="m-a7" class="usa-accordion__content usa-prose">
+    TK   
+  </div>
+
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a8"
+    >
+      If you're using a custom Gulp workflow
+    </button>
+  </h4>
+  <div id="m-a8" class="usa-accordion__content usa-prose">
+    TK   
+  </div>
+
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a9"
+    >
+      If you're using Webpack
+    </button>
+  </h4>
+  <div id="m-a9" class="usa-accordion__content usa-prose">
+    TK   
+  </div>
+</div>
+
+### 5. Update to Sass module syntax
```

Comment:
@bonnieAcameron Should we include the [Optional] tag that we have in the TOC in the titles for steps 5-7?

Reply:
Yes, thank you @amyleadem !

Reply:
Should these be added to the subnav as well?

![image](https://user-images.githubusercontent.com/25211650/164094091-252a77ac-ceb7-43a5-b0d4-5b9702eca921.png)

Reply:
So adding the brakets to the subnav text seems to break the site! I'll add it to the section headers though!

Reply:
Added the tags similar to how Dan did in the overview.

Resolved in https://github.com/uswds/uswds-site/pull/1529/commits/f77aab9eacaa30043a17050eba2ba43e1ebddc61

![image](https://user-images.githubusercontent.com/25211650/164103585-ab120d5e-9ac4-4b24-bbb9-f39bb8c99de7.png)

![image](https://user-images.githubusercontent.com/25211650/164103487-9014f519-0066-4c70-9a83-24d8349d1f55.png)

![image](https://user-images.githubusercontent.com/25211650/164103514-1369bc37-ef80-43b2-8507-d4bfbb089490.png)

![image](https://user-images.githubusercontent.com/25211650/164103544-15d79e13-a44e-47af-bc8d-0afdb4323935.png)